product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD is based on a global Raft model where clusters have a single Raft group spanning all the data nodes in the PGD cluster. A Raft leader is elected by the cluster and propagates the state of the cluster to all the other nodes in the cluster. 
```

Comment:
"PGD manages metadata using a global Raft model..." ? 

Worried that is "based on wording" will make reader think replication is using Raft (as there are similar systems which do that).

Reply:
Fixed